core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -173,8 +172,7 @@ public PGConnectionContext(
             int workerCount
     ) {
         this.engine = engine;
-        this.utf8Sink = new DirectCharSink(engine.getConfiguration().getTextConfiguration().getUtf8SinkSize());
```

Comment:
lets restore sink to be part of the "connected session" (these sessions are pooled and reused, so not huge memory hog)